qe_metrics/utils/product_utils.py

```diff
@@ -28,3 +28,21 @@ def products_from_file(products_file: str) -> List[Dict[Any, Any]]:
             LOGGER.error(f"Error occurred parsing queries for product {name}: {err}")
     orm.commit()
     return products
+
+
+def append_last_updated_arg(query: str, look_back_days: int) -> str:
+    """
+    Add the last updated argument to the queries.
+
+    Args:
+        query (str): Query to be modified
+        look_back_days (int): Number of days to look back when querying for issues
+
+    Returns:
+        str: A query with an argument to filter issues that have been updated in the last 'look_back_days' days
+    """
+    if any(word in query for word in ["updatedDate", "updated"]):
+        LOGGER.error("Query is already using the 'updatedDate' or 'updated' field. Please remove it:\n{query}")
+        return query
```

Comment:
we want to block this query I guess, or strip it.
we shouldn't get to this point since the queries will be reviewed.

Reply:
I don't know how we could strip it, as we don't know where the argument for `updatedDate` or `updated` ends in the query. I have just modified this to raise a `ValueError`. This stops the query from being executed, but allows any subsequent queries to still execute. Like you said, this should not get past the review process.